### PR TITLE
Add Howl manifest

### PR DESCRIPTION
**Pull request by refi64**

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build io.howl.Editor

**Comment by barthalion** [comment]
Build failed on aarch64: https://flathub.org/builds/#/builders/21/builds/193

Have you contacted upstream developer about this pull request?

**Comment by refi64** [comment]
@barthalion I just added a flathub.json to skip aarch64, apparently that's not a particularly well-tested part of LuaJIT.

> Have you contacted upstream developer about this pull request?

I *am* part of upstream. :wink: (But yeah either way, the main developer OK'd this, and Howl already has a Flatpak manifest in their Git repo.)

**Comment by barthalion** [comment]
Sorry, back in a day it was single person project.

File: io.howl.Editor.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+app-id: io.howl.Editor
+runtime: org.gnome.Platform
+runtime-version: '3.32'
+sdk: org.gnome.Sdk
+command: howl
+rename-appdata-file: howl.appdata.xml
+rename-desktop-file: howl.desktop
```
Comment (barthalion) [inline io.howl.Editor.yaml]: Renames should ideally happen upstream.

**Comment by barthalion** [comment]
bot, build io.howl.Editor

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/io.howl.Editor

Let me know if I should add more people. Thanks!

**Pull request closed without merging**